runtime/src/bank.rs

```diff
@@ -1144,6 +1144,11 @@ pub struct NewBankOptions {
     pub vote_only_bank: bool,
 }
 
+#[derive(Debug, Default)]
+pub struct BankTestConfig {
+    pub enable_secondary_indexes: bool,
```

Comment:
Would be more useful to have this take an `AccountSecondaryIndexes`, then a bool setting everything on or off.